be/src/common/config.h

```diff
@@ -472,7 +412,7 @@ CONF_String(buffer_pool_clean_pages_limit, "20G");
 CONF_mInt64(memory_maintenance_sleep_time_s, "10");
 
 // Alignment
-CONF_Int32(memory_max_alignment, "16");
+CONF_mInt32(memory_max_alignment, "16");
```

Comment:
I think you can remove this config, and define it as a static const value, there is no reason to update it.

Reply:
OK.